notes/src/App.js

```diff
@@ -0,0 +1,128 @@
+import React, { Component } from 'react';
+import './App.css';
+import axios from 'axios';
+import { Route } from 'react-router-dom';
+import NotesList from './components/NotesList';
+import SingleNote from './components/SingleNote';
+import SideBar from './components/SideBar';
+import CreateNote from './components/CreateNote';
+import EditNote from './components/EditNote';
+
+class App extends Component {
+  constructor() {
+    super()
+    this.state = {
+      notes: [],
+      title: '',
+      textBody: '',
+      updatedTitle: '',
+      updatedText: '',
+      deleting: false
+    }
+  }
+
+  componentDidMount() {
+    axios
+      .get('https://fe-notes.herokuapp.com/note/get/all')
+      .then(res => {
+        this.setState({ notes: res.data })
+      })
+      .catch(err => {
+        console.log('Error: ', err)
+      })
+  }
+
+  handleInput = e => {
+    this.setState({ [e.target.name]: e.target.value })
+  }
+
+  handleSubmit = () => {
+    axios
+      .post('https://fe-notes.herokuapp.com/note/create', {
+        title: this.state.title,
+        textBody: this.state.textBody
+      })
+      .then(response => {
+        console.log(response.data)
+      })
+      .catch(error => {
+        console.log('Error: ', error);
+      });
+      this.setState({
+          title: '',
+          textBody: ''
+        });
+  };
+
+  deleteItem = id => {
+    axios
+      .delete(`https://fe-notes.herokuapp.com/note/delete/${id}`)
+      .then(response => {
+        console.log(response)
+      })
+      .catch(error => console.log(error));
+      this.setState();
+  };
+
+  toggleDeletingOn = () => {
+    this.setState({ deleting: true });
+  };
+  toggleDeletingOff = () => {
+    this.setState({ deleting: false });
+  };
+
+  editingNote = (title, textBody) => {
+    this.setState({ updatedTitle: title, updatedText: textBody });
+  };
+
+   editedNote = id => {
```

Comment:
This is not a good name for your method; consider something like `submitEditedNote` or something that will tell future viewers what this method does